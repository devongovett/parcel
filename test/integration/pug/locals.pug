doctype html
html(lang="en")
  body
    #container.col
      if youAreUsingPug
        p You are amazing
      else
        p Get on it!
