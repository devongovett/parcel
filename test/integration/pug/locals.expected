<!DOCTYPE html>
<html lang="en">
  <body>
    <div class="col" id="container">
      <p>You are amazing</p>
    </div>
  </body>
</html>